lib/orderbook/MatchingEngine.ts

```diff
@@ -12,6 +12,7 @@ type SplitOrder = {
 };
 
 type orderId = string;
+type PeerPubKey = string;
```

Comment:
Not a big deal at all but I'm not sure we need a type alias for `PeerPubKey`. I didn't know we had an `orderId` type actually, I don't think we're using it in most places. My main concern is that using it inconsistently would be bit confusing.